src/components/StatsGrid/StatsGridConfig.tsx

```diff
@@ -70,10 +77,44 @@ export const columnDefs = [
 
       return <span className={className}>{params.value}</span>;
     },
+    tooltipComponent: HTMLTooltip,
+    headerTooltip: (() => {
+      const states = [
+        ["Active", "Node is active and accepting new connections."],
+        [
+          "Draining",
+          "Node is being parepared for a software update by draining connections (waiting for existing connections to finish and not accepting any new connections) to restart gracefully. This is a temporary state, the node will become active again after the update.",
+        ],
+        [
+          "Inactive",
+          "Node has not updated its status for a while and has been removed from the network until it reports its status again.",
+        ],
+        [
+          "Down",
+          "Node has been failing health checks and has been removed from the network until it reports successful health checks again. It is also possible that node with that status has been manually blocked by the network operator for malicious activities.",
```

Comment:
I would be more explicit and say that the node has failed `at least one` health check in the recent minutes. The way is worded right now might exaggerate how _down_ it is

Reply:
```suggestion
          "Node has failed at least one health check in recent minutes and has been removed from the network until it reports successful health checks again. It is also possible that node with that status has been manually blocked by the network operator for malicious activities.",
```